src/base/bittorrent/session.cpp

```diff
@@ -3028,6 +3031,20 @@ void Session::setAnnounceToAllTiers(bool val)
     }
 }
 
+int Session::asyncIOThreads() const
+{
+    return qMax(m_asyncIOThreads.value(), 1);
```

Comment:
Shouldn't it have max value limit? If no, why do you limit it in GUI?

Reply:
>If no, why do you limit it in GUI?

The default for spinbox is 99, I am raising it in reality.